tests/quality/test_pydantic_models.py

```diff
@@ -0,0 +1,89 @@
+from enum import Enum
```

Comment:
I'll check in additional test cases once we confirm that the overall approach here makes sense